Assets/Scripts/Achievements/AchievementManager.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Assets.Scripts.Characters.Titan;
+using Assets.Scripts.Events.Args;
+using Assets.Scripts.Services;
+using Assets.Scripts.Services.Interface;
+using UnityEngine;
+
+namespace Assets.Scripts.Achievements
+{
+    public class AchievementManager : MonoBehaviour
+    {
+        private readonly IPlayerService playerService = Service.Player;
+
+        private void Start()
+        {
+            playerService.OnTitanDamaged += OnTitanDamage;
+        }
+
+        private void OnDestroy()
+        {
+            playerService.OnTitanDamaged -= OnTitanDamage;
+        }
+
+        /// <summary>
+        /// This is for damage related achievements.
+        /// </summary>
+        /// <param name="ev"></param>
+        public void OnTitanDamage(TitanDamagedEvent ev)
+        {
+            switch (ev.Titan.Type)
+            {
+                case TitanType.Female:
+                case TitanType.Armored:
+                case TitanType.Beast:
+                case TitanType.Colossal:
+                case TitanType.Eren:
+                case TitanType.Mindless:
+                    if (ev.Damage >= 1000)
+                    {
+                        UnlockAchievement(Achievement.DMG_1K);
+                    }
+                    else return;
```

Comment:
Remove these "else return" statements, as you cannot obtain the 2K achievement.

Reply:
As discussed on Discord (Leaving the comment explaining why it works for future reference) if the player deals 2K damage, it would still pass the check as it's over 1K, so it does still work! 😄